### DISCUSSION TITLE
The easy part is wiring, the hard part is figuring out where you went wrong.  LOL!

### DISCUSSION THREAD

**Discussion by ej012345** [picocomputer/community#8] [Q&A]
![ej pico computerV1b 0](https://github.com/picocomputer/discussions/assets/131208326/81df9d1e-d320-4fbb-92c0-fe36fd719121)
The lights are on, but nobody's home.

**Comment by ixmeins** [comment]
At least it looks impressive :-) I hope to finish my veroboard version today.

**Comment by rumbledethumps** [comment]
I added a few testing notes to the DIY post:
https://github.com/orgs/picocomputer/discussions/4

**Reply by ej012345** [reply]
Yes… Thank you! Progress
![IMG_2569](https://github.com/picocomputer/discussions/assets/131208326/45d84cc9-7b88-4a4c-9b27-2494179c4140)

**Comment by ej012345** [comment]
![IMG_2570](https://github.com/picocomputer/discussions/assets/131208326/53b70b25-1311-4d37-afa8-453e2f469322)
